config/platforms/platform_intelsgx.h.in

```diff
@@ -0,0 +1,74 @@
+#include <endian.h>
+#include <stdint.h>
+#include <sys/types.h>
+#include <mbusafecrt.h>
+
+/*
+ * Intel SGX enclaves are completely isolated from the underlying
+ * kernel and therefore syscalls are not allowed by default. This
+ * means that getting the current time and locale information is
+ * impossible unless there is intervention by the enclave developers.
+ *
+ * Thus enclave developers must define DUK_USE_DATE_GET_NOW and
+ * DUK_USE_DATE_GET_LOCAL_TZOFFSET for their enclave either manually
+ * in duk_config.h or via other means (other config header, compiler
+ * flags). If these are not defined compilation will fail with a
+ * preprocessor error.
```

Comment:
This should just say "in duk_config.h", no other means are supported. So something like:
> Enclave developers must provide DUK_USE_DATE_GET_NOW and DUK_USE_DATE_GET_LOCAL_TZOFFSET when configuring Duktape for build.